### PR TITLE
top bar added

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes issue #62 
-Changes: Added top bar in login and signup page.
sample deployment -> http://hushed-whip.surge.sh/
This is the first step. Will be pushing more to make it look exactly same.

### CODE REVIEW

**Comment by DravitLochan** [comment]
Please help me resolve travis build fail error as well. 
Thanks

File: src/components/Auth/Login/Login.react.js [hunk 1]
```diff
@@ -205,9 +205,20 @@ class Login extends Component {
...
 		};
 		return (
-			<div className="loginForm">
-				<Paper zDepth={0} style={styles}>
-					<h1>Login to SUSI</h1>
+			<div>
+				<div>
+                	<header className='message-thread-heading'>
+                    	<AppBar
+                        	className="app-bar"
```
Comment (madhavrathi) [inline src/components/Auth/Login/Login.react.js]: add this here
 iconElementLeft={<IconButton></IconButton>}
we do not want any hamburger icon here, other than this everything looks good.
Thanks

Comment (DravitLochan) [inline src/components/Auth/Login/Login.react.js]: Sure

**Comment by DravitLochan** [comment]
@rishiraj824 ok

**Comment by DravitLochan** [comment]
@madhavrathi @isuruAb @rishiraj824 @uday96 I have removed the hamBurger as asked. Also builds passed when I tested locally!

**Comment by DravitLochan** [comment]
@mariobehling Please merge this! Thanks

**Pull request merged**